code/index.html

```diff
@@ -3,45 +3,133 @@
+        <div class="game-score-box">
+          <h2 class="modal-heading primary-heading">Game</h2>
+
+          <div class="best-score-container">
+            <h3>Best Score</h3>
+          <div id="best-score-container-inner">
+          
+          </div>
+          </div>
+          <table class="modal-table">
+            <thead>
+              <tr>
+                <th colspan="3">The Game Results</th>
+              </tr>
+            </thead>
+         
+              <tr>
+                <th>Player Name</th>
+                <th>How Many times you guessed</th>
+                <th>Time</th>
+              </tr>
+              <tbody id="table-body">
+             <!-- JS create elements  -->
+               
+            
+            </tbody>
+          </table>
+          <button class="game-clear-btn">Clear Data</button>
+        </div>
+        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 1440 320" class="modal-svg"><path fill="#e5ff9c" fill-opacity="1" d="M0,128L40,160C80,192,160,256,240,250.7C320,245,400,171,480,160C560,149,640,203,720,208C800,213,880,171,960,170.7C1040,171,1120,213,1200,224C1280,235,1360,213,1400,202.7L1440,192L1440,320L1400,320C1360,320,1280,320,1200,320C1120,320,1040,320,960,320C880,320,800,320,720,320C640,320,560,320,480,320C400,320,320,320,240,320C160,320,80,320,40,320L0,320Z"></path></svg>
+      </div>
+     
+      </svg>
+    </div>
+
+    <!-- From here is the main page -->
+
     <aside class="question-section">
       <button type="button" id="restart" class="outlined-button">RESTART</button>
 
-      <img
-        class="guess-who-icon"
-        src="images/guess-who-bubble.png"
-        alt="Guess Who"
-      />
+      <img class="guess-who-icon" src="images/guess-who-bubble.png" alt="Guess Who" />
 
-      <h1>Does the person have</h1>
+      <h1 class="primary-heading">Does the person have</h1>
 
       <select title="questions" id="questions">
         <optgroup label="hair">
           <option value="brown">brown hair</option>
           <option value="yellow">yellow hair</option>
-          <!-- Add the rest of hair colors as options  -->
+          <option value="black">black hair</option>
+          <option value="white">white hair</option>
+          <option value="orange">orange hair</option>
+          <option value="hidden">hidden hair</option>
+          <option value="purple">purple hair</option>
+          <option value="grey">grey hair</option>
+          <!-- hair colors as options  -->
         </optgroup>
         <optgroup label="eyes">
           <option value="hidden">hidden eyes</option>
-          <!-- Add the rest of eye colors as options  -->
+          <option value="green">green eyes</option>
+          <option value="brown">brown eyes</option>
+          <option value="blue">blue eyes</option>
+          <option value="black">black eyes</option>
+          <!-- eye colors as options  -->
+        </optgroup>
+        <optgroup label="wedlock">
```

Comment:
I liked the added options for the questions, they're fun and make the people on the cards more alive. But they doesn't tell the player anything, or help them find out who the secret person is? There's no way the player can tell who is married and who is single. Maybe this is the point of the questions, to be like "trick questions"?
I still think it's a cool festure though!